edx_proctoring/tests/test_reviews.py

```diff
@@ -175,6 +178,34 @@ def test_psi_review_callback(self, psi_review_status, review_status, credit_requ
             else:
                 self.assertEqual(len(notifications), 0)
 
+
+    def test_psi_video_url(self):
+        """
+        Test that review callback from PSI produces encrypted video link
+        """
+
+        with patch.object(SoftwareSecureBackendProvider, 'get_video_review_aes_key') as key_mock:
```

Comment:
Not an easy way to mock out the backend settings in a way that actually affects the provider class, as the provider class is initialized upon app load. In order to get the desired aes key value, I had to mock the getter.